### DISCUSSION TITLE
MinEmacs logo

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#81] [Ideas]
Hi.

Do you have any logo such as this:

![image](https://github.com/abougouffa/minemacs/assets/17734314/a7886cd2-eddf-402c-9b98-485e788b5671)

But for MinEmacs:

![image](https://github.com/abougouffa/minemacs/assets/17734314/fb71c352-eb06-4724-850e-14f39e4ed71f)

**Comment by abougouffa** [comment]
Nop, I didn't make an icon for it (as it is just a configuration, not a fork of Emacs!)
